code/game/machinery/flasher.dm

```diff
@@ -125,10 +128,9 @@
 	anchored = !anchored
 	if(anchored)
 		WRENCH_ANCHOR_MESSAGE
-		overlays.Cut()
 	else
 		WRENCH_UNANCHOR_MESSAGE
-		overlays += "[base_state]-s"
+	update_icon()
```

Comment:
```suggestion
	update_icon(UPDATE_OVERLAYS)
```
dont you only want to update the overlays?

Reply:
For unanchoring it, it also changes the icon_state along with the overlay so I don't think I should only be overlays